File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/ChatSessionCreateForm.kt
package com.sdkwork.app

data class ChatSessionCreateForm(
    val name: String? = null,
    val description: String? = null,
    val modelId: String? = null,
    val systemPrompt: String? = null,
    val temperature: Double? = null,
    val maxTokens: Int? = null,
    val topP: Double? = null,
    val type: String? = null,
    val resourceId: String? = null
)

File: sdkwork-app-sdk-go/types/chat_session_create_form.go
package types

// 创建聊天会话请求
type ChatSessionCreateForm struct {
	Name string `json:"name"`
	Description string `json:"description"`
	ModelId string `json:"modelId"`
	SystemPrompt string `json:"systemPrompt"`
	Temperature float64 `json:"temperature"`
	MaxTokens int `json:"maxTokens"`
	TopP float64 `json:"topP"`
	Type string `json:"type"`
	ResourceId string `json:"resourceId"`
}
